### PR TITLE
Update directory_helper.php

### PR DESCRIPTION
**Pull request by churavka**
Return only array directory names witout DIRECTORY_SEPARATOR.

### CODE REVIEW

**Comment by narfbg** [comment]
Elaborate please.

**Comment by vlakoff** [comment]
As already discussed in the past, we add a trailing separator after the directories because it's the only way to distinguish them from files.

File: system/helpers/directory_helper.php [hunk 1]
```diff
@@ -80,11 +80,9 @@ function directory_map($source_dir, $directory_depth = 0, $hidden = FALSE)
 					continue;
 				}
 
-				is_dir($source_dir.$file) && $file .= DIRECTORY_SEPARATOR;
-
 				if (($directory_depth < 1 OR $new_depth > 0) && is_dir($source_dir.$file))
 				{
-					$filedata[$file] = directory_map($source_dir.$file, $new_depth, $hidden);
+					$filedata[$file] = directory_map($source_dir.$file.DIRECTORY_SEPARATOR, $new_depth, $hidden);
```
Comment (vlakoff) [inline system/helpers/directory_helper.php]: This change is actually useless

File: system/helpers/directory_helper.php [hunk 2]
```diff
@@ -80,11 +80,9 @@ function directory_map($source_dir, $directory_depth = 0, $hidden = FALSE)
 					continue;
 				}
 
-				is_dir($source_dir.$file) && $file .= DIRECTORY_SEPARATOR;
```
Comment (vlakoff) [inline system/helpers/directory_helper.php]: As commented on the PR, this should be kept.

**Pull request closed without merging**